priv/beam.json

```diff
@@ -142,7 +142,7 @@
       "pluginVersion": "7.1.3",
       "targets": [
         {
-          "expr": "beam_memory_total_kilobytes{job=\"$job\", instance=\"$instance\"}",
+          "expr": "beam_memory_allocated_kilobytes{job=\"$job\", instance=\"$instance\"}",
```

Comment:
`bytes` is a Prometheus convention: https://prometheus.io/docs/practices/naming/